### ISSUE TITLE
Colab version?

### ISSUE THREAD

**Issue by EtianAM** [akai-katto/dandere2x#124]
Hi, I know it might not be that easy, but it would be really helpful to have it. I tried to do it myself, but I could not. The best I could do was to compile [waifu2x-ncnn-vulkan](https://drive.google.com/open?id=1xxLQofw_pK-zAsU3NUcJCuopEzNmgbDw), but when I executed it, it would mark an error starting. If you could make a version or instruct me to run it I would appreciate it.

**Comment by Tremeschin** [comment]
Hi, you don't need to compile the stuff unless you're using Linux (not that it'd be hard to do so, there is a old documentation about setting it up if you're into this OS)

Check out the releases tab from the main page of the repository we have all the binaries ready to be used packed up in a release :)

**Comment by EtianAM** [comment]
Colab run in Ubuntu. I follow the instructions, but I was unable to succes.
Thanks for answer

**Comment by Tremeschin** [comment]
Oh, Ubuntu is a distro we had troubles running Dandere2x on, to be honest we have yet to find a easy way we can write a guide on it.

IIRC there's an waifu2x-ncnn-vulkan available from the snap (some users got working with that), and you should already have the base packages for compiling the C++ part of it..

Though if you compiled waifu2x vulkan by yourself and want to run it make sure its executable is pointed right in the yaml config file.

Ah, and don't forget the graphic drivers and vulkan loaders / headers (dunno the packages on Ubuntu sorry)

Let me know how it goes!!

**Comment by EtianAM** [comment]
Well, I tried to do it with waifu2x-ncnn-vulkan, but I'm not lucky, something is missing and I haven't been able to find out what, it still doesn't do anything. But... I was able to run waifu2x_cafe, following the video2x guidelines and running a ubuntu version (modifying some files to work with Colab graphics and deleting characters from Shift JIS). Now, follow the Dandere2x run, but I doubt it will be easy.
